go.mod

```diff
@@ -0,0 +1,73 @@
+module github.com/Mellanox/nvidia-k8s-ipam
+
+go 1.20
+
+require (
+	github.com/go-logr/logr v1.2.4
+	github.com/spf13/cobra v1.7.0
+	k8s.io/api v0.26.4
+	k8s.io/apimachinery v0.26.4
+	k8s.io/client-go v0.26.4
+	k8s.io/component-base v0.26.4
+	k8s.io/klog/v2 v2.90.1
+	sigs.k8s.io/controller-runtime v0.14.6
```

Comment:
controller-runtime is used to get LeadeElection, HealthChecks and some other useful stuff for free.